src/uct/sm/mm/base/mm_iface.c

```diff
@@ -57,6 +57,10 @@ ucs_config_field_t uct_mm_iface_config_table[] = {
      "Size of the FIFO element size (data + header) in the MM UCTs.",
      ucs_offsetof(uct_mm_iface_config_t, fifo_elem_size), UCS_CONFIG_TYPE_UINT},
 
+    {"FIFO_MAX_WINDOW", UCS_PP_MAKE_STRING(UCT_MM_IFACE_FIFO_MAX_WINDOW),
```

Comment:
maybe MAX_POLL
(window is usually number of outstanding uncompleted operations)

Reply:
done